lcd/emonPiLCD.py

```diff
@@ -354,7 +355,7 @@ def __init__(self, logger):
                     open('/home/pi/data/emonpi', 'a').close()
                 break
 
-        if lcd_status.rstrip() == 'False':
+        if not lcd_status:
```

Comment:
`check_output` was returning a string, either the string "True" or the string "False", but both are boolean True, so this never run the body.